rum-demo-app-ios/RUM Mobile Demo App/Common/Html/sample1.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html><html><head>
```

Comment:
How are these html files generated? Or where do they come from?

Reply:
sample1.html & sample2.html - These two files were used to load inside inapp webview. 
We updated it and current code is not using these two files. So I am removing these two files from the code.